packages/back-end/types/visual-changeset.d.ts

```diff
@@ -3,6 +3,8 @@ interface DOMMutation {
   action: "append" | "set" | "remove";
   attribute: string;
   value: string;
+  parentSelector?: string;
```

Comment:
Should `value` be optional here too since it's not used all the time?

Reply:
Updated